### ISSUE TITLE
Feature request: Nx.dev compatibility

### ISSUE THREAD

**Issue by binary64** [allegro/typescript-strict-plugin#32]
Hi I love the idea of your plugin. I'm trying to get it to work in my monorepo, which uses Nx.dev.

Here's how you can get a repo locally to test with:
```bash
npm i -g yarn && \
rm -rf ~/nx-demo && \
cd ~/ && \
npx create-nx-workspace nx-demo --packageManager=yarn --style=less --nx-cloud=false --interactive=false --preset=empty --linter=eslint && \
cd nx-demo
```

I currently get this error
```
Error: Command failed with exit code 1: tsc --showConfig
error TS5081: Cannot find a tsconfig.json file at the current directory: C:/p/nx_test.
```

Many thanks!

**Comment by kamkry** [comment]
Hi, you can quickly fix it either by renaming `tsconfig.base.json` to `tsconfig.json`, or by adding new `tsconfig.json` with following body:
```json
{
  "extends": "./tsconfig.base.json",
}

```

**Comment by binary64** [comment]
I tried both of your suggests, but I run `node_modules/.bin/update-strict-comments` I get:
```
Project does not contain any strict files.
```
Any ideas? It's a pretty plain Nx repo

**Comment by kamkry** [comment]
Does your project include any typescript files? That error is thrown only when either there's no TS files or every TS file is ignored with `//@ts-strict-ignore` comment

**Comment by binary64** [comment]
Yes, I only have mostly .tsx and .ts files. And a couple of .js files.

I don't have any @ts-strict-ignore's

**Comment by anatter** [comment]
@binary64 I currently have the same issue, have you found a solution yet?

**Comment by TheBox193** [comment]
Same here. Looks like this plugin lacks support for entry points for projects with multiple tsconfig files.

**Comment by gabrielbryk** [comment]
Any update on this? Would like to use this plugin but running into the same issue

**Comment by fomnn** [comment]
same here
